common/src/View/ScaleObjectsToolPage.cpp

```diff
@@ -42,39 +42,89 @@ namespace TrenchBroom {
         m_document(document) {
             createGui();
         }
-        
+
+        void ScaleObjectsToolPage::activate() {
+            const auto document = lock(m_document);
+            const auto suggestedSize = document->hasSelectedNodes() ? document->selectionBounds().size() : vm::vec3::zero;
```

Comment:
I'm not sure that zero is a good default. Or would that never happen anyway?

Reply:
It'll happen if you activate the tool with nothing selected. Maybe just an empty text box would be better in that case than "0 0 0"?

Reply:
Should it be possible to even activate the tool in that case? I forget, how does it work with other tools?

Reply:
Oh yeah, you can't even activate the tool when there's no selection. So the vm::vec3::zero case should never happen.